README.md

```diff
@@ -81,9 +81,21 @@ Run `carthage update` to build the framework and drag the built `WeScan.framewor
 
 Just download the project, and drag and drop the "WeScan" folder in your project.
 
+
+<p align="center">
+    <img width="900px" src="Assets/project.png">
+</p>
+
+
+Simply Add the WeScan FrameWork into project `Embedded Binaries` and `Linked Frameworks and Libraries`.
+
+<p align="center">
+    <img width="900px" src="Assets/LinkedFrameworks.png">
+</p>
+
 ## Usage
 
-1. Make sure that your ViewController conforms to the `ImageScannerControllerDelegate` protocol
+1. Make sure that your ViewController confirms to the `ImageScannerControllerDelegate` protocol
```

Comment:
Objects conform to protocols.
For example to quote [this document](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/WorkingwithProtocols/WorkingwithProtocols.html): 
>This chapter describes the syntax to define a formal protocol, and explains how to mark a class interface as conforming to a protocol,

Reply:
This still hasn't been fixed @ITHHKN

Reply:
Also, it seems like you accidentally removed the `### Swift` title